Update Dockerfile and bump GLIBC_VERSION to 2.25

sam-caldwell opened the pull request:
Bumping glibc version to 2.25

sam-caldwell committed:
Update Dockerfile and bump GLIBC_VERSION to 2.25

Bumping glibc version to 2.25

sam-caldwell commented:
sdurrheimer, since 2.25 is released, I thought I would bump the version here so the docker image will pass security scans complaining about vulnerabilities since 2.23.

sdurrheimer commented on Dockerfile:
```diff
@@ -1,7 +1,7 @@
 MAINTAINER  Steve Durrheimer <s.durrheimer@gmail.com>
 
-ENV GLIBC_VERSION="2.23-r1"
+ENV GLIBC_VERSION="2.25-r1"
```
`2.25-r0` given that `2.25-r1` is still a pre-release.

sdurrheimer reviewed (changes requested):
Thanks you !

Could you also change the github repo url given that it has been moved to `sgerrand/alpine-pkg-glibc`.

sam-caldwell committed:
Update Dockerfile (git repo and glibc version)

Changes as requested.
(1) changed 2.25 from r1 to r0 (...thanks s.durrheimer for pointing out my mistake)
(2) Made GIT_REPO a variable so it is easier to change in the future.

sam-caldwell commented:
Change was submitted as requested.

sdurrheimer commented:
Sorry for the delay.
